src/api/v1/media.ts

```diff
@@ -29,9 +29,12 @@ export async function postMedia(c: Context<{ Variables: Variables }>) {
   if (file.type.startsWith("video/")) {
     imageBytes = await makeVideoScreenshot(imageData);
   }
-  const image = sharp(imageBytes);
-  const fileMetadata = await image.metadata();
-  const content = new Uint8Array(imageData);
+
+  const image = sharp(imageBytes).rotate();
+  const rmMetaImage = await image.toBuffer();
```

Comment:
You may want to keep the color profile, so:
```suggestion
  const rmMetaImage = await image.keepIccProfile().toBuffer();
```